CONTRIBUTING.md

```diff
@@ -12,12 +12,12 @@ document and take heed of its wisdom.
 
 ## Pull Requests
 
-This project uses Travis to check that it always builds on all the supported
-platforms. However, Microsemi's IT department stubbornly refuses (for silly
-reasons) to turn it on for the main repository. Therefore, in order to ensure
-your contributions get tested, please submit all pull requests to
-[lsgunth/switchtec-user][2]. From there we will ensure your contribution
-gets merged into the main repository.
+Please set the base branch to microsemi/devel when create the pull request.
+Make sure the comparing changes inclue your bug fixes or new features.
+It is highly recommended that check the patch by checkpatch.pl script of
+Linux Kernel tree, confirm no obvious style problems and is ready for
+submission. Once the patch go through our review and verification stages,
+it will be merged into microsemi/devel branch.
```

Comment:
The english in every one of these sentences has gotten pretty broken. I suggest you get someone to edit it for you.

Also, you didn't add the description of the style we follow as I suggested in my previous review.